src/main/resources/sql/defaultResources/port_data_production.sql

```diff
@@ -960,3 +960,322 @@ INSERT INTO widgetcatalog (code, titles, parameters, plugincode, parenttypecode,
 <property key="contentType">NWS</property>
 <property key="modelId">10021</property>
 </properties>', 0, NULL);
+
+INSERT INTO pagemodels (code,descr,frames,plugincode,templategui) VALUES ('homepage1','Homepage 1','<?xml version="1.0" encoding="UTF-8"?>
+<frames>
+  <frame pos="0">
+      <descr>Logo</descr>
+      <sketch x1="0" y1="0" x2="2" y2="0" />
+  </frame>
+  <frame pos="1">
+      <descr>Navigation bar</descr>
+      <sketch x1="3" y1="0" x2="5" y2="0" />
+  </frame>
+  <frame pos="2">
+      <descr>Search</descr>
+      <sketch x1="6" y1="0" x2="8" y2="0" />
+  </frame>
+  <frame pos="3">
+      <descr>Login</descr>
+      <sketch x1="9" y1="0" x2="11" y2="0" />
+  </frame>
+  <frame pos="4">
+      <descr>Breadcrumb</descr>
+      <sketch x1="0" y1="1" x2="11" y2="1" />
+  </frame>
+  <frame pos="5">
+      <descr>Banner</descr>
+      <sketch x1="0" y1="2" x2="11" y2="2" />
+  </frame>
+  <frame pos="6">
+      <descr>News</descr>
+      <sketch x1="0" y1="3" x2="11" y2="3" />
+  </frame>
+  <frame pos="7">
+      <descr>Frame 4 Colonne 16</descr>
+      <sketch x1="0" y1="4" x2="11" y2="4" />
+  </frame>
+  <frame pos="8">
+      <descr>Footer 1 Colonne 16</descr>
+      <sketch x1="0" y1="5" x2="11" y2="5" />
+  </frame>
+  <frame pos="9">
+      <descr>Footer 2 Colonne 16</descr>
+      <sketch x1="0" y1="6" x2="11" y2="6" />
+  </frame>
+</frames>
+
+',NULL,'<#assign wp=JspTaglibs["/aps-core"]>
+
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title>
+            <@wp.currentPage param="title" />
+        </title>
+        <meta name="viewport" content="width=device-width,  user-scalable=no" />
+        <link rel="icon" href="<@wp.info key="systemParam" paramName="applicationBaseURL" />favicon.png" type="image/png" />
+        <!-- Custom OOTB page template styles -->
+        <link rel="stylesheet" href="<@wp.resourceURL />static/css/ootb/page-templates/index.css" rel="stylesheet">
+        <!-- Carbon Design System -->
+        <link rel="stylesheet" href="https://unpkg.com/carbon-components/css/carbon-components.min.css">
+        </head>
+        <body>
+          <div class="Homepage1__header">
```

Comment:
```suggestion
          <div class="Homepage1 Homepage__header">
```
better for semantics